src/core/components/array-model.jsx

```diff
@@ -15,7 +15,7 @@ export default class ArrayModel extends Component {
   }
 
   render(){
-    let { getComponent, required, schema, depth, expandDepth, name } = this.props
```

Comment:
`required` passed into the `<Model>` in render was hard coded to `false` and the red '*' after the model was removed because it was moved to be after the property name a couple weeks back. So the passed-in `required` prop was no longer used in the render method.